ups/meas_pz.table

```diff
@@ -3,6 +3,7 @@
 # - Common third-party packages can be assumed to be recursively included by
 #   the "base" package.
 setupRequired(base)
+setupRequired('testdata_rail')
```

Comment:
Should be `testdata_rail` without quotes

Reply:
Thanks, did this get fixed in main when Dan merged his version?

Reply:
I removed `testdata_rail` usage entirely in main. You're welcome to rebase and keep it on this branch, or keep it in the new package with the other unsupported estimators.